GVFS/GVFS.Service/Handlers/RequestHandler.Windows.cs

```diff
@@ -0,0 +1,37 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using GVFS.Service.Handlers;
+using System.Runtime.Serialization;
+
+namespace GVFS.Service.Handlers
+{
+    public class WindowsRequestHandler : RequestHandler
+    {
+        private const string EnablePrjFltRequestDescription = "attach volume";

```

Comment:
`EnablePrjFltRequestDescription` should get moved to the base class with the other descriptions (now that the base class handles the message as a no-op).

I'd also suggest renaming this to something like `EnableProjFSRequestDescription` so that it's not Windows specific.